src/hydra/dataProvider.js

```diff
@@ -467,7 +467,7 @@ export default (
               apiSchema = api;
               return { data: api, customRoutes };
             })
-            .catch(({ error, status }) => {
+            .catch((error, status) => {
```

Comment:
This is because `apiDocumentationParser` does not throw an object with error and status fields.

Reply:
I've never seen a catch with 2 arguments, imo it should be something like:

```
catch((error) => {
  const status = error.status
  // then check later if needed
})
```

Reply:
that's how it was before 2.2.1, but somehow `status` was added. 
https://github.com/api-platform/admin/compare/v2.2.0...v2.2.1#diff-00342c75206eee49e87735785d9c2e74L470